packages/website/src/components/EditorTabs.tsx

```diff
@@ -25,17 +24,17 @@ export default function EditorTabs({
             <button
               role="tab"
               className={styles.tabStyle}
-              key={item}
-              aria-selected={activeTab === item}
-              disabled={activeTab === item}
-              onClick={(): void => change(item)}
+              key={item.label}
+              aria-selected={activeTab === item.value}
+              disabled={activeTab === item.value}
+              onClick={(): void => change(item.value)}
             >
-              {item}
+              {item.label}
```

Comment:
These code changes don't look like they actually fix the issue? Is there a benefit to them?